src/ai/composite/goal.cpp

```diff
@@ -138,7 +138,7 @@ void target_unit_goal::add_targets(std::back_insert_iterator< std::vector< targe
 	for (const unit &u : resources::gameboard->units()) {
 		if (ufilt( u )) {
 			LOG_AI_GOAL << "found explicit target unit at ... " << u.get_location() << " with value: " << value() << "\n";
-			*target_list = target(u.get_location(), value(), target::TYPE::EXPLICIT);
+			*target_list = target(u.get_location(), value(), ai_target::type::XPLICIT);
```

Comment:
…why'd you drop the E here?

Reply:
`explicit` is a C++ keyword, so the lowercase naming wouldn't work.

Reply:
I wanted to keep the lowercase and uppercase names the same.